packages/nibijs-examples/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@nibiruchain/nibijs-examples",
+  "dependencies": {
+    "@nibiruchain/nibijs": "../nibijs"
+  },
+  "devDependencies": {
+    "ts-node": "^10.9.1",
+    "bun": "^1.0.7",
```

Comment:
Is bun actually being used here?

We use yarn package-wide (and so do most js devs), and would prefer it for consistency, also it is not very compatible with windows yet; https://bun.sh/docs/installation#:~:text=Bun%20provides%20a%20limited%2C%20experimental%20native%20build%20for%20Windows.

Reply:
I can remove it. I stopped using node entirely about 3 months ago, running `bun test` or `bun install` instead of `yarn test` and everything, for example. 

Can remove it since Windows support is bad